contrib/backup/multicloud/auth/auth.go

```diff
@@ -0,0 +1,37 @@
+// Copyright 2019 The OpenSDS Authors.
```

Comment:
fileter is used for api-server client, please remove this code.

Reply:
sorry，this is used for keystone.go, I can not remove this code.